scripts/test_coverage.sh

```diff
@@ -11,7 +11,7 @@ export CARGO_INCREMENTAL=0
 export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=3 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests"
 export RUSTDOCFLAGS=""
 rm -vf ./target/**/*.gcda
-cargo build
-cargo test --lib
+cargo build --features "test-srs, gadgets, bls, schnorr"
+cargo test --lib --features "test-srs, gadgets, bls, schnorr"
```

Comment:
couldn't we include things like `gadgets, bls, schnorr` in the `default` so that we don't have to explicitly list them here? 

it's not super clear which one should I list. 

I also don't know why couldn't we use `--all-features` here, remind me of the problem again?

Reply:
`--all-features` may turn on `iclcle`